data/scripts/actions/christmas_bundle.lua

```diff
@@ -0,0 +1,42 @@
+local christmasBundle = Action()
+
+function christmasBundle.onUse(player, item, fromPosition, target, toPosition, isHotkey)
+	local presents = {
+		[6507] = { -- red christmas bundle
```

Comment:
a comment should include the purpose of the mapping, eg.:
-- (christmas present item id) => ((reward item id, count), ...)